### PR TITLE
[Flink AI Flow] Quickstart.md

### PR DESCRIPTION
**Pull request by jiangxin369**
## What is the purpose of the change

Make quickstart work on refactored code.

## Brief change log

*(for example:)*
  - *Send StopDagEvent to Airflow scheduler via notification service*
  - *Once received StopDagEvent, Airflow scheduler kills all running dag runs*

## Verifying this change

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

### CODE REVIEW

**Comment by SteNicholas** [comment]
@jiangxin369 , why is the database backend of AIFlow diferent from the database of Airflow in Quick Start? IMO, the database of AIFlow could be MySQL.

File: flink-ai-flow/QUICKSTART.md [hunk 1]
```diff
@@ -2,117 +2,108 @@
...
+Web Server pid: 21861
+Master Server log: ${AIRFLOW_HOME}/logs/master_server.log
+Master Server pid: 21862
 Airflow deploy path: ${AIRFLOW_HOME}/airflow_deploy
 Visit http://127.0.0.1:8080/ to access the airflow web server.
 ```
 If you see the login page when visit http://127.0.0.1:8080/, you can login with the defaut user name(admin) and password(admin).
+This is the scheduler web UI which you can see the workflows that you summitted.
 
-### Prepare AI Flow Project
```
Comment (SteNicholas) [inline flink-ai-flow/QUICKSTART.md:77]: It's recommended to introduce the structure of the project.

File: flink-ai-flow/QUICKSTART.md [hunk 2]
```diff
@@ -2,117 +2,108 @@
...
-# This path should be determined by yourself
+# 1. Because the default scheduler service is Apache Airflow, so we need to set ${AIRFLOW_HOME}, this path could be determined by yourself.
 export AIRFLOW_HOME=~/airflow
 AIRFLOW_DEPLOY_PATH="${AIRFLOW_HOME}/airflow_deploy"
 
-# This path is the absolute path of the flink-ai-flow project in your machine
+# 2. The example is prepared in ${SOURCE_CODE_DIR}/examples/demo, ${SOURCE_CODE_DIR} is the absolute path 
+# of the flink-ai-flow project. You can directly copy it to your working directory.
 SOURCE_CODE_DIR=/tmp/flink-ai-extended/flink-ai-flow
+cp -r $SOURCE_CODE_DIR/examples/demo $AIRFLOW_HOME/
```
Comment (SteNicholas) [inline flink-ai-flow/QUICKSTART.md:93]: Why does the demo directory move to `AIRFLOW_HOME`?

File: flink-ai-flow/QUICKSTART.md [hunk 3]
```diff
@@ -2,117 +2,108 @@
...
 ```
 
-## Install AI Flow
-Currently the AI Flow bundles a modified Airflow so you do not need to install the Apache Airflow manually.
-We added an event-based scheduler named event_scheduler to Apache Airflow which is more powerful and has a Web UI to monitor the execution.
+## Get Started
+### Install AI Flow
 
-If you are installing AI Flow from source, you can install AI Flow by running the following command:
+If you are installing AI Flow from source, you could easily run following commands:
```
Comment (SteNicholas) [inline flink-ai-flow/QUICKSTART.md:41]: Where is the way that install Flink AI Flow from PyPi?

**Review by SteNicholas — CHANGES_REQUESTED** [review]

File: flink-ai-flow/bin/start_aiflow.py [hunk 4]
```diff
@@ -27,18 +27,25 @@ def create_default_sever_config(root_dir_path):
...
-        notification_uri: localhost:50052
+
+        # whether to start the scheduler service
+        start_scheduler_service: True
+
+        # scheduler config
+        scheduler:
+          scheduler_class_name: ai_flow_plugins.scheduler_plugins.airflow.airflow_scheduler.AirFlowScheduler
+          scheduler_config:
+            airflow_deploy_path: /tmp/airflow
```
[the remark this answers is not in the document]
Comment (jiangxin369) [reply flink-ai-flow/bin/start_aiflow.py:47]: IMO this config is related to specific scheduler, we'd better address it in scheduler plugin doc.

File: flink-ai-flow/bin/start_aiflow.py [hunk 5]
```diff
@@ -27,18 +27,25 @@ def create_default_sever_config(root_dir_path):
...
+            notification_service_uri: localhost:50052
```
[the remark this answers is not in the document]
Comment (jiangxin369) [reply flink-ai-flow/bin/start_aiflow.py:48]: IMO this config is related to specific scheduler, we'd better address it in scheduler plugin doc.

File: flink-ai-flow/QUICKSTART.md (hunk 1 above)
Comment (jiangxin369) [reply flink-ai-flow/QUICKSTART.md:77]: IMO it should be introduced in tutorial doc by @bgeng777

File: flink-ai-flow/QUICKSTART.md (hunk 2 above)
Comment (jiangxin369) [reply flink-ai-flow/QUICKSTART.md:93]: Modified

**Pull request closed without merging**